libs/initScripts/manageProdRelayerWallets.js

```diff
@@ -0,0 +1,232 @@
+    // const gasPrice = "0x1e08ffca00"
+    // fast price = 0x8d8f9fc00
+    // new fast price = 0x2e90edd000
+    // const gasPrice = "0x2e90edd000"
+    // const gasPrice = "0x1e80355e00"
+    // from identity - 0xf7100
+    const gasLimit = "0xf7100"
+    const nonce = await web3.eth.getTransactionCount(address)
+    console.log(`Sending tx from ${sender.publicKey} to ${receiverAddress} with nonce=${nonce}, gasPrice=${gasPrice}, gasLimit=${gasLimit}`)
+    let txParams = {
+        nonce: web3.utils.toHex(nonce),
+        gasPrice: gasPrice,
+        gasLimit: gasLimit,
+        to: receiverAddress,
+        value: web3.utils.toHex(value)
+    }
+    const tx = new EthereumTx(txParams)
+    tx.sign(privateKeyBuffer)
+    const signedTx = '0x' + tx.serialize().toString('hex')
+    try {
+        const receipt = await web3.eth.sendSignedTransaction(signedTx)
+    } catch(e) {
+        console.log(`Failed with ${e}`)
+    }
+}
+
+const fundEthRelayerIfEmpty = async () => {
+    let walletInfo = await loadProdRelayerWallets()
+
+    let relayerWallets = walletInfo.relayerWallets
+
+    let gasInfo
+    let gasPrice
+
+    for (let i = 0; i < relayerWallets.length; i++) {
+        let relayerPublicKey = relayerWallets[i].publicKey
+        let balance = await ethWeb3.eth.getBalance(relayerPublicKey)
+        let validBalance = parseInt(balance) >= minimumBalance
+        console.log(`${i + 1} - Found balance ${balance} for ${relayerPublicKey}, validBal=${validBalance}`)
+        if (!validBalance) {
+            let missingBalance = minimumBalance - balance
+            console.log(`${i + 1} - Funding ${relayerPublicKey} with ${missingBalance}, currently ${balance}/${minimumBalance}`)
+            gasInfo = await getGasPrice()
+            gasPrice = gasInfo.fastestGweiHex
+            await createAndSendTransaction(
+                walletInfo.funder, // Always send from the designated FUNDER
+                relayerPublicKey,             // Public key of receiving account
+                missingBalance,     // Min
+                ethWeb3,
+                gasPrice
+            )
+           console.log(`${i + 1} - Finished Funding ${relayerPublicKey} with ${minimumBalance}`)
+           balance = await ethWeb3.eth.getBalance(relayerPublicKey)
+           console.log(`${i + 1} - Updated balance ${balance} for ${relayerPublicKey}`)
+        }
+    }
+    await queryAccountBalances(relayerWallets)
+}
+
+let args = process.argv
+const run = async () => {
+    switch (args[2]) {
+        case 'fundAllRelayers':
+            await fundEthRelayerIfEmpty()
+            break
+        case 'queryRelayerBalances':
+            await queryRelayerBalances()
+            break
+        case 'generatenewAccounts':
+            await generateNewAccounts()
+            break
+        default:
+            throw new Error("Invalid argument")
+    }
+}
+
+run()
+
+/*
+const calculateProdRelayDistribution = async () => {
```

Comment:
can delete this i don't think we will need to perform this operation moving fwd

Reply:
deleted